shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-distsql/shardingsphere-encrypt-distsql-parser/src/main/java/org/apache/shardingsphere/encrypt/distsql/parser/core/EncryptDistSQLStatementVisitor.java

```diff
@@ -88,7 +88,8 @@ public ASTNode visitEncryptColumnDefinition(final EncryptColumnDefinitionContext
                 getIdentifierValue(ctx.cipherColumnDefinition().dataType()),
                 null == ctx.plainColumnDefinition() ? null : getIdentifierValue(ctx.plainColumnDefinition().dataType()),
                 null == ctx.assistedQueryColumnDefinition() ? null : getIdentifierValue(ctx.assistedQueryColumnDefinition().dataType()),
-                (AlgorithmSegment) visit(ctx.algorithmDefinition()));
+                (AlgorithmSegment) visit(ctx.algorithmDefinition()),
```

Comment:
How about support distsql in another PR？Is it error occurred?

Reply:
ok